### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
UTCs:

- Validated that all the following actions are working as expected

1. GET SPECIFIC CVE DETAILS
2. ADVANCE CVE SEARCH
3. CVE SEARCH BY KEYWORDS
4. GET CVE CHANGE HISTORY
5. CPE SEARCH

### CODE REVIEW

**Review by cs-omkar-kumbhar — APPROVED** [review]

File: nist-nvd/info.json [hunk 1]
```diff
@@ -0,0 +1,1082 @@
...
+                ],
+                "value": "HIGH"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Use Search Flags",
+          "name": "useSearchFlags",
+          "description": "Uses Search Flags",
```
Comment (cs-prerna-joshi) [inline nist-nvd/info.json]: update the description.

File: nist-nvd/info.json [hunk 2]
```diff
@@ -0,0 +1,1082 @@
...
+            "true": [
+              {
+                "title": "Start Date",
+                "name": "pubStartDate",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "type": "datetime",
+                "tooltip": "Publish Start Date",
+                "description": "Start Date and time for Publish Date"
```
Comment (cs-prerna-joshi) [inline nist-nvd/info.json]:
```suggestion
                "description": "Specify the Start Date and time for Publish Date for CVE."
```

File: nist-nvd/info.json [hunk 3]
```diff
@@ -0,0 +1,1082 @@
...
+              },
+              {
+                "title": "End Date",
+                "name": "pubEndDate",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "type": "datetime",
+                "tooltip": "Publish End Date",
+                "description": "End Date and time for Publish Date"
```
Comment (cs-prerna-joshi) [inline nist-nvd/info.json]:
```suggestion
                "description": "Specify the End Date and time for Publish Date for CVE."
```

File: nist-nvd/info.json (hunk 1 above)
Comment (cs-rohit-khune) [reply nist-nvd/info.json]: Updated Description

File: nist-nvd/info.json (hunk 2 above)
Comment (cs-rohit-khune) [reply nist-nvd/info.json]: Updated description

File: nist-nvd/info.json (hunk 3 above)
Comment (cs-rohit-khune) [reply nist-nvd/info.json]: Updated description

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**